src/modules/billing/services/job-service.js

```diff
@@ -37,6 +37,15 @@ const setEmptyBatch = (eventId, batchId) =>
 const setFailedJob = (eventId, batchId) =>
   setStatuses(eventId, jobStatus.error, batchId, BATCH_STATUS.error);
 
+/**
+* Sets the event/job to error, and the batch to error
+* @param {String} eventId
+* @param {String} batchId
+*/
+const setReviewJob = (eventId, batchId) =>
+  setStatuses(eventId, jobStatus.review, batchId, BATCH_STATUS.review);
```

Comment:
I think we need to address how the events are used here possibly in a future ticket/refinement

Reply:
I've made a list of things of possible bugs/things that might need changing, I'll add it to that list. The waiting page relies on the event status and changing that felt too big and too out of scope of this ticket.